starlite/parsers.py

```diff
@@ -67,11 +67,11 @@ def parse_form_data(media_type: "RequestEncodingType", form_data: "FormMultiDict
         if not isinstance(value, MultipartUploadFile):
```

Comment:
The LGTM tool is complaining about an unused import on line 14, but removing the import (or marking it with `# noqa` is causing the flake8 pre-commit check to fail (because the import is used on line 48).

If anyone has suggestions for how to resolve this I would be grateful!

Reply:
Pycharm has this problem too:

![image](https://user-images.githubusercontent.com/20659309/191626377-51640822-abb8-4850-aab6-c3f6442f32ea.png)

Their inspections mustn't be wired to destringify the first argument to cast.

I think the flake8 is working properly and problem with lgtm.

There is an lgtm ignore `# lgtm[py/unused-import]`([ref](https://lgtm.com/rules/6770079/), [ref](https://lgtm.com/help/lgtm/alert-suppression#python))

Reply:
See their note though:

> If you've [enabled](https://lgtm.com/help/lgtm/managing-automated-code-review#enabling-pr-integration) automated code review of pull requests for your projects, it's important to note that alert suppression changes made in a pull request are only applied when that pull request is merged. So, if you suppress an alert in a pull request, LGTM will continue to report the related alert for as long as the pull request is open. For more information, see [Automated code review FAQs](https://lgtm.com/help/lgtm/automated-code-review-faqs#why-automated-code-review-reports-misses-alert).

Reply:
I guess I'll leave it alone then!